src/components/OffcanvasAdvancedFilters.tsx

```diff
@@ -42,11 +42,11 @@ export default function OffcanvasAdvancedFilters({ closeOffcanvas }: OffcanvasPr
         </div>
       </DOffcanvasHeader>
       <DOffcanvasBody>
-        <div className="d-flex flex-column gap-4 pt-3">
+        <div className="d-flex flex-column gap-6 pt-4">
           <DDatePicker
```

Comment:
```suggestion
          <DDatePicker<never, true>
```

Para habilitar los rangos hay que ponerle el 2do parametro en `true`